docs/2_4_settings.md

```diff
@@ -32,9 +32,9 @@ ReVanced Manager has settings that can be configured to your liking.
 
   Manage the keystore used to sign patched apps.
 
-- ### 📄 Export, import or reset ReVanced Patches selection
+- ### 📄 Export, import or reset ReVanced Patch selections
 
-  Manage the ReVanced Patches selection. This is useful if you want to share your ReVanced Patches selection with others or reset it to the default selection.
+  Manage the ReVanced Patch selections. This is useful if you want to share your ReVanced Patch selections with others or reset it to the default selection.
```

Comment:
I am unsure if it should be changed to `...default selections.`

I think it is more correct, but not appropriate for this context.